android/src/main/java/com/rnappauth/RNAppAuthModule.java

```diff
@@ -63,6 +63,7 @@
     private final ReactApplicationContext reactContext;
     private Promise promise;
     private boolean dangerouslyAllowInsecureHttpRequests;
+    private Double connectionTimeoutMillis;
     private Boolean skipCodeExchange;
```

Comment:
Used a Double instead of Integer because I saw this in the docs
<img width="300" alt="Screen Shot 2022-01-20 at 7 20 03 PM" src="https://user-images.githubusercontent.com/13589602/150459868-e997a729-9140-4abc-b11c-c585b728e384.png">